HarfangLab/harfanglab/ingest/parser.yml

```diff
@@ -185,5 +195,63 @@ stages:
           event.type: ["info"]
           event.provider: "{{json_event.message.source_name}}"
           event.code: "{{json_event.message.event_id|string}}"
-          action.properties: '{ {% for name, value in json_event.message.event_data.items() %}{% if value != None %}"{{name.replace(" ","")}}": "{{value | replace("\n", "") | replace("\\", "\\\\")}}",{% endif %}{% endfor %} }'
+          action.properties: "{{json_event.message.event_data}}"
           action.id: "{{json_event.message.event_id}}"
+
+  authentication_info:
+    actions:
+      - set:
+          event.category: ["authentication"]
+          event.type: >
+            {% set type_value %}
+              {% if json_event.message.event_type == "login_failure" %}
+                end
+              {% elif json_event.message.event_type == "login" %}
+                start
+              {% else %}
+                info
+              {% endif %}
+            {% endset %}
+            {{ type_value.replace('\n', '').replace(' ', '') }}
```

Comment:
I suggest using a `translate` action to map `event.type`. That should avoid all the replacement instructions.